exporter/stackdriverexporter/stackdriver.go

```diff
@@ -92,20 +76,38 @@ func newStackdriverMetricsExporter(cfg *Config) (exporter.MetricsExporter, error
 		exporterhelper.WithShutdown(mExp.Shutdown))
 }
 
-func getStackdriverOptions(ProjectID, MetricPrefix string) stackdriver.Options {
+func newStackdriverExporter(cfg *Config) (*stackdriver.Exporter, error) {
 	// TODO:  For each ProjectID, create a different exporter
 	// or at least a unique Stackdriver client per ProjectID.
-
-	return stackdriver.Options{
+	options := stackdriver.Options{
 		// If the project ID is an empty string, it will be set by default based on
 		// the project this is running on in GCP.
-		ProjectID: ProjectID,
+		ProjectID: cfg.ProjectID,
 
-		MetricPrefix: MetricPrefix,
+		MetricPrefix: cfg.Prefix,
 
 		// Set DefaultMonitoringLabels to an empty map to avoid getting the "opencensus_task" label
 		DefaultMonitoringLabels: &stackdriver.Labels{},
 	}
+	if cfg.Endpoint != "" {
+		dOpts := []grpc.DialOption{}
+		if cfg.Insecure {
```

Comment:
Suggestion: document somewhere that `Insecure` will only have effect when using a custom endpoint.

Reply:
Done.